server/index.js

```diff
@@ -166,6 +167,10 @@ app.post('/login', (req, res, done) => {
 		successRedirect: '/'
 	})(req, res, done);
 });
+app.get('/logout', authenticated, (req, res) => {
```

Comment:
You should use `post` method instead of `get`, as browsers will prefetch these routes, and thus log the user out automatically